sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -26,10 +25,10 @@
     DEFAULT_ENTITY_DF_EVENT_TIMESTAMP_COL,
     _get_requested_feature_views_to_features_dict,
 )
-from feast.protos.feast.core.DataSource_pb2 import DataSource as DataSourceProto
 from feast.registry import Registry
 from feast.repo_config import FeastConfigBaseModel, RepoConfig
-from feast.value_type import ValueType
+
+from .bigquery_source import BigQuerySource
```

Comment:
This import is suspect. Should it be `feast.bigquery_source`?

Reply:
No, it's a relative import - should work fine (tests are happy too)

Reply:
I can't remember where this has caused problems for me, but I distinctly remember always having to make all imports absolute. Somewhere I ran into problems with relative imports. I don't think there are any relative imports in the code base, except the __init__.py files.

I'm fine leaving it as is and seeing if a problem pops up though.